package.json

```diff
@@ -36,14 +36,15 @@
     "@aws-sdk/lib-dynamodb": "^3.341.0",
     "@aws-sdk/util-dynamodb": "^3.362.0",
     "@types/luxon": "^3.3.0",
+    "ajv": "^8.12.0",
```

Comment:
seems slightly annoying to pull in the `ajv` dependency for just some typing... Out of curiousity, does it work to import it as a dev dependency?